Remote SSH: "cannot find nc"

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When trying to open a remote folder, I get an error: 

> **Failed to connect**
> Cannot find nc, which is required to connect over ssh.. Please try again. 

I have no idea what "nc" is. Google offers no help. I've tried with multiple servers and immediately get the same error. 

### Zed Version and System Specs

Zed: v0.165.4 (Zed)
OS: Linux X11 manjaro unknown
Memory: 31.3 GiB
Architecture: x86_64
GPU: AMD Radeon RX 7900 GRE (RADV NAVI31) || radv || Mesa 24.2.8-arch1.1

### If applicable, add screenshots or screencasts of the incorrect state / behavior

![image](https://github.com/user-attachments/assets/17d032f9-357d-4679-a177-2112e595cb99)


### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary>

<!-- Click below this line and paste or drag-and-drop your log-->
```
2025-01-06T23:36:32.520779756-05:00 [ERROR] Cannot find nc, which is required to connect over ssh.
```
<!-- Click above this line and paste or drag-and-drop your log--></details>

--- a/crates/remote/src/ssh_session.rs
+++ b/crates/remote/src/ssh_session.rs
@@ -1434,7 +1434,7 @@ impl SshRemoteConnection {
 
         anyhow::ensure!(
             which::which("nc").is_ok(),
-            "Cannot find nc, which is required to connect over ssh."
+            "Cannot find `nc` command (netcat), which is required to connect over SSH."
         );
 
         // Create an askpass script that communicates back to this process.

